frontend/amundsen_application/static/js/components/OwnerEditor/index.tsx

```diff
@@ -248,14 +248,25 @@ export class OwnerEditor extends React.Component<
     // check if rendering an owner category that lacks any entries
     let isEmptySection = false;
 
-    if (section) {
+    console.log(`itemProps: ${JSON.stringify(itemProps)}`);
+
+    // if any owners uncategorized, set section to null to render an uncategorized owners list
+    Object.keys(itemProps).forEach((key) => {
+      if (!itemProps[key].additionalOwnerInfo?.owner_category) {
+        section = null;
+      }
+    });
```

Comment:
is this needed if you do the other check before this function is called? this feels like it might be more complicated than it needs to be

Reply:
Yeah it wasn't needed both places, I deleted it from here.